src/Hazelcast.Net.Examples/Hazelcast.Net.Examples.csproj

```diff
@@ -39,12 +39,4 @@
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
   </ItemGroup>
 
-  <!--
```

Comment:
What was this part doing?

Reply:
sort-of re-including parts of the `Testing` project for use in `Examples` and that was... not exactly pretty, not doing it anymore